pkg/controllers/cloud/configmap_controller.go

```diff
@@ -30,11 +30,38 @@ import (
 	"antrea.io/nephe/pkg/controllers/config"
 )
 
+type ConfigMapHandler func(controllerConfig *config.ControllerConfig)
+
+type controllerConfigType int
```

Comment:
Add a comment here saying why and what is the enum.

Reply:
Solved.